src/Standards/Squiz/Sniffs/Commenting/FunctionCommentThrowTagSniff.php

```diff
@@ -51,16 +57,18 @@ protected function processTokenWithinScope(File $phpcsFile, $stackPtr, $currScop
         $find[] = T_WHITESPACE;
 
         $commentEnd = $phpcsFile->findPrevious($find, ($currScope - 1), null, true);
-        if ($tokens[$commentEnd]['code'] === T_COMMENT) {
-            // Function is using the wrong type of comment.
-            return;
-        }
+        if ($this->skipFunctionsWithoutDocComment === true) {
+            if ($tokens[$commentEnd]['code'] === T_COMMENT) {
+                // Function is using the wrong type of comment.
+                return;
+            }
 
-        if ($tokens[$commentEnd]['code'] !== T_DOC_COMMENT_CLOSE_TAG
-            && $tokens[$commentEnd]['code'] !== T_COMMENT
-        ) {
-            // Function doesn't have a doc comment.
-            return;
+            if ($tokens[$commentEnd]['code'] !== T_DOC_COMMENT_CLOSE_TAG) {
+                // Function doesn't have a doc comment.
+                return;
+            }
+        } else if ($tokens[$commentEnd]['code'] !== T_DOC_COMMENT_CLOSE_TAG) {
+            $commentEnd = $currScope;
```

Comment:
The `else if` statement is added here - we use `$commentEnd` to add errors, in case when we do not skip functions without doc comment and we are processing function without doc comment errors are going to be added on `$currScope` which is `function`.